index.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * @file index.js
+ *
+ * Copyright (c) 2020 Simon Fraser University
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
```

Comment:
I'd suggest including a GPLv3 `LICENSE` file in the repo, then specifying `For full terms see the file LICENSE.` (We've been recently moving that way with our repos too.)